src/pages/index.js

```diff
@@ -141,3 +254,41 @@ profileEditButton.addEventListener("click", () => {
 addNewCardButton.addEventListener("click", () => {
   popupAddWithForm.open();
 });
+
+editAvatarButton.addEventListener("click", () => {
+  popupAvatarEditForm.open();
+});
+
+//deleteCardButton.addEventListener("click", () => {
+//popupDeleteWithForm.open();
+//}
+// instantiate API
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "4e1d8109-9ece-45e8-8ccd-12c7e4a219df",
+    "Content-Type": "application/json",
+  },
+});
+
+// api
+//   .getInitialCards()
+//   .then((data) => {
+//     // data will be an array with 1 card in it
+//   })
+//   .catch((err) => {});
+
+// api
+//   .getUserInfo()
+//   .then((data) => {
+//     // data will be an array with 1 card in it
+//   })
+//   .catch((err) => {});
+
+Promise.all([api.getInitialCards(), api.getUserInfo()]).then(
+  ([cards, userData]) => {
+    section.renderItems(cards);
+    userInfo.setUserInfo(userData.name, userData.about);
+    userInfo.setAvatarImage(userData);
+  }
+);
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere in the code**